doc/specs/File-diff-spec.md

```diff
@@ -0,0 +1,41 @@
+# New PowerToys utility: File comparison tool
+
+<img align="right" height="200px" src="https://github.com/microsoft/PowerToys/blob/user/aaron-junker/diff-spec/doc/images/icons/PowerToys%20icon/PNG/1440x2160.png" />
+
+- **What is it:** File comparison tool
+- **Authors:** [Aaron Junker](https://github.com/aaron-junker)
+- **Spec Status:** Draft
+- **GitHub issue:** [#14950](https://github.com/microsoft/PowerToys/issues/14950)
+
+## 1 Overview
+
+This utility will add a tool to view differences between two files.
+
+### 1.1 Technical implementation

```

Comment:
Tech is a 2nd point, user scenarios is first.  This should be possible investigation points, not assume it is done